docs/rest.md

```diff
@@ -671,7 +671,7 @@ Generally the UUID of a background operation you are interested in.
 * Description: Create a local user
 * Access: trusted
 * Operation: sync
-* Return: an object with the created username and the amount of imported ssh keys
+* Return: an object with the created username, the ssh keys imported and the amount of imported ssh keys
```

Comment:
Good point, thanks!

I've added this as an entry in our tech debt micro tasks so we can merge this now.